### PR TITLE
U/fe/1

### PR DESCRIPTION
**Pull request by frossie**
First draft just to see if it builds

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,17 +1,360 @@
...
-############################################
+On JIRA workflows for observatory operations
+============================================
 
 Abstract
-========
+--------
 
-.. abstract::
+.. container:: abstract
```
Comment (jonathansick) [inline index.rst]: Why did you switch the abstract directive for a container directive?

Comment (frossie) [reply index.rst]: I didn't

Comment (frossie) [reply index.rst]: I did roundrip index.rst to index.md and then back again with pandoc when I couldn't get md to work so maybe some stuff happened tnen (ie not a normal situation)

File: README.rst [hunk 2]
```diff
@@ -10,9 +10,9 @@ On JIRA workflows for ovservatory operations
 RTN-066
 =======
 
-Rubin has extensive and in some cases sophisticated workflows based on JIRA for anything from managing planned work, to reporting bugs,  to defining validation campaigns,  to managing staff travel. There are 12 highly active JIRA projects supporting this diverse collection of workflows and they are central to the working day of project staff. 
+Rubin has extensive and in some cases sophisticated workflows based on JIRA for anything from managing planned work, to reporting bugs,  to defining validation campaigns,  to managing staff travel. There are 12 highly active JIRA projects supporting this diverse collection of workflows and they are central to the working day of project staff.
```
Comment (Slimleashma) [inline README.rst:13]: Remove extra spaces between commas and clauses.

File: obs_jira.png
Comment (Slimleashma) [inline obs_jira.png:1]: Could you change the word "master" to "umbrella" or "principle"

**Review by Slimleashma — COMMENTED** [review]
Cheers Frossie. I'm here for it. Thanks for writing this up.

**Pull request merged**